source/apiVolontaria/volunteer/tests/tests_model_Cell.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework.test import APIClient, APITestCase
+
+from apiVolontaria.factories import UserFactory, AdminFactory
+from location.models import Address, StateProvince, Country
+from ..models import Cell
+
+
+class CellTests(APITestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+        self.user = UserFactory()
+        self.user.set_password('Test123!')
+        self.user.save()
+
+        self.admin = AdminFactory()
+        self.admin.set_password('Test123!')
+        self.admin.save()
+
+        self.random_country = Country.objects.create(
+            name="random country",
+            iso_code="RC",
+        )
+        self.random_state_province = StateProvince.objects.create(
+            name="random state",
+            iso_code="RS",
+            country=self.random_country,
+        )
+        self.address = Address.objects.create(
+            address_line1='random address 1',
+            postal_code='RAN DOM',
+            city='random city',
+            state_province=self.random_state_province,
+            country=self.random_country,
+        )
+
+    def test_create_cell(self):
```

Comment:
Add two other test to be sure we can't create a cell without one of this `required` arguments.

Reply:
Done. However, we can still create a cell with only an `address`. That is because the charfield default is an empty string. We can't change this behaviour on the DB level unless we override the save() and update() methods in th model.